netadapter/router/route.go

```
@@ -61,7 +64,7 @@ func (r *Route) EnqueueWithTimeout(message wire.Message, timeout time.Duration)
 	}
 	select {
 	case <-time.After(timeout):
-		return false, errors.New("timeout expired")
+		return false, ErrTimeout
```

Dequeue should return this error as well.